MetaMask/metamask-extension
# fix: for realease blocker bugs

## **Description**
With the new async ppom changes, we don't see any more the metrics information for any Signature (eth_sign, typed signature) and we are now also missing the ppom request count to Infura for Transactions and Signatures. We should see it in Transaction/Signature Confirmed/Rejected.

## **Related issues**

Fixes: #22840 #22841 

## **Manual testing steps**

1. Enable Blockaid
2. Trigger any Signature/Transaction
3. See Metrics
4. Accept/Reject the Signature/Transaction
5. See Metrics in network info

## **Screenshots/Recordings**



### **Before**

![302900276-b316bcdc-a574-4a60-92db-dc0e4692fc9e-2](https://github.com/MetaMask/metamask-extension/assets/44811/e666d6b5-8095-4b84-8247-c7d6b4385917)

![302901962-355ce267-f234-4373-abe3-17310724618d-2](https://github.com/MetaMask/metamask-extension/assets/44811/d5f1c2c1-1a50-43c3-927d-c1cf78bfcdd6)


### **After**

<img width="1222" alt="Screenshot 2024-02-08 at 14 08 51" src="https://github.com/MetaMask/metamask-extension/assets/44811/75fd3d07-76a6-404d-856b-ac757169824f">
<img width="1227" alt="Screenshot 2024-02-08 at 14 08 28" src="https://github.com/MetaMask/metamask-extension/assets/44811/7511a78c-0ec3-4840-b851-6693e81aa67e">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
